src/docs/user/references/Variables.adoc

```diff
@@ -187,6 +187,14 @@ For example:
 
 [[_dynamic_variables]]
 
+=====  Icon management variables
+
+There are specific workflow variables corresponding to icons.
```

Comment:
This is somewhat incoherent because task icons are not workflow variables.
You could say for example: 
- There are specific workflow [variables] and task variables [that are] dedicated to icons (or icon management).
- There are specific variables that are dedicated to icon management.

Reply:
Thanks for good comments!